<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineWriter</class>
 <widget class="QDialog" name="LineWriter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Write Lines</string>
  </property>
  <widget class="QLabel" name="label_Line">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>32</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Line to Write:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="text_Submitted">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>191</y>
     <width>601</width>
     <height>271</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Submit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>154</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>601</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Write">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>55</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_PageN">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Page 1/1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_LineN">
   <property name="geometry">
    <rect>
     <x>553</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Line 1/5</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_EnterLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter your line:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_CurrentPage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Page:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Discard">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>154</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Discard Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Abort">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>154</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
